lib/puppet/indirector/node/json.rb

```diff
@@ -0,0 +1,30 @@
+require 'puppet/node'
+require 'puppet/indirector/json'
+
+class Puppet::Node::Json < Puppet::Indirector::JSON
+  include Puppet::Util::SymbolicFileMode
```

Comment:
Since the yaml node terminus didn't support symbolic file modes, I'm inclined to not support it here. If we did that, then I don't think we need to override any methods in this terminus, as the superclass will "do the right thing"